go.mod

```diff
@@ -41,6 +41,7 @@ require (
 	github.com/shirou/gopsutil v2.18.10+incompatible
 	github.com/sirupsen/logrus v1.2.0
 	github.com/spaolacci/murmur3 v0.0.0-20180118202830-f09979ecbc72
+	github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2
```

Comment:
why add this?

Reply:
I don't know how this happened.
same as master expect parser version

Reply:
It disappears after merging the master branch..